CRM/ManualDirectDebit/Upgrader.php

```diff
@@ -142,6 +142,9 @@ public function install() {
     $this->createDirectDebitPaymentInstrument();
     $this->createDirectDebitPaymentProcessorType();
     $this->createDirectDebitPaymentProcessor();
+    
+    // Reset columnSpecs cache to avoid re-creating fields on log tables.
```

Comment:
can you in the comment explain why we do this and the issue that happen when the line is not there for reference